docs/route.md

```diff
@@ -125,94 +125,46 @@ This is a simple property, so you can both read it and write to it.
 
 ##### m.route.Link
 
-This component can create a dynamic routable link:
+This component creates a dynamic routed link. Its essential function is to produce `a` links with local `href`s transformed to take account of the [route prefix](#mrouteprefix).
 
 ```javascript
-m(m.route.Link, {href: "/test"})
-```
-
-Using `m.route.Link` causes the link to behave as a router link - clicking it navigates to the route specified in `href`, instead of navigating away from the current page to the URL specified in `href`.
+m(m.route.Link, {href: "/foo"}, "foo")
 
-You can also set the `options` passed to `m.route.set` when the link is clicked by passing the `options` attribute:
-
-```javascript
-m(m.route.Link, {href: "/test", options: {replace: true}})
+// Renders to
+// <a href="#!/foo">foo</a>
```

Comment:
This is true if the prefix is `#`, might be good to say "If the route prefix is set to "#" that would produce the following html:"

Reply:
Addressed in update.